common/faults/faults.c

```diff
@@ -234,49 +208,39 @@ void handleCallbacks(fault_status_t recievedStatus) {
     }
 }
 
-//BEGIN AUTO RECIEVE FUNCTIONS
-void fault_sync_main_module_CALLBACK(CanParsedData_t *msg_header_a) {
-	fault_status_t recievedStatus = {msg_header_a->fault_sync_main_module.latched, msg_header_a->fault_sync_main_module.idx};
-	handleCallbacks(recievedStatus);
-}
-void fault_sync_driveline_CALLBACK(CanParsedData_t *msg_header_a) {
-	fault_status_t recievedStatus = {msg_header_a->fault_sync_driveline.latched, msg_header_a->fault_sync_driveline.idx};
-	handleCallbacks(recievedStatus);
-}
-void fault_sync_dashboard_CALLBACK(CanParsedData_t *msg_header_a) {
-	fault_status_t recievedStatus = {msg_header_a->fault_sync_dashboard.latched, msg_header_a->fault_sync_dashboard.idx};
-	handleCallbacks(recievedStatus);
-}
-void fault_sync_precharge_CALLBACK(CanParsedData_t *msg_header_a) {
-	fault_status_t recievedStatus = {msg_header_a->fault_sync_precharge.latched, msg_header_a->fault_sync_precharge.idx};
-	handleCallbacks(recievedStatus);
-}
-void fault_sync_torque_vector_CALLBACK(CanParsedData_t *msg_header_a) {
-	fault_status_t recievedStatus = {msg_header_a->fault_sync_torque_vector.latched, msg_header_a->fault_sync_torque_vector.idx};
-	handleCallbacks(recievedStatus);
-}
-void fault_sync_test_node_CALLBACK(CanParsedData_t *msg_header_a) {
-	fault_status_t recievedStatus = {msg_header_a->fault_sync_test_node.latched, msg_header_a->fault_sync_test_node.idx};
-	handleCallbacks(recievedStatus);
-}
-//END AUTO RECIEVE FUNCTIONS
-
-//Force faults from daq
-void set_fault_CALLBACK(CanParsedData_t *msg_header_a) {
-    if (GET_OWNER(msg_header_a->set_fault.id) == currentMCU) {
-        forceFault(msg_header_a->set_fault.id, msg_header_a->set_fault.value);
+/**
+ * @brief Implement Force fault requests from DAQ
+ *
+ * @param id ID of fault to force
+ * @param value State to set fault to
+ *
+ * @return none
+ */
+void set_fault_daq(uint16_t id, bool value) {
+    if (GET_OWNER(id) == currentMCU) {
+        forceFault(id, value);
     }
 }
 
-//Return control back to this mcu from daq
-void return_fault_control_CALLBACK(CanParsedData_t *msg_header_a) {
-    if (GET_OWNER(msg_header_a->set_fault.id) == currentMCU) {
-        unForce(msg_header_a->set_fault.id);
+/**
+ * @brief Return control back to this mcu from daq
+ *
+ * @param id ID of fault to return
+ *
+ * @return none
+ */
+void return_fault_control(uint16_t id) {
```

Comment:
Can you go through and set functions that aren't supposed to be called externally from the library to static. This also applies to variables.